plugins/node/readme.md

```diff
@@ -0,0 +1,34 @@
+# @dotcom-tool-kit/node
+
+Run your Node application.
```

Comment:
**question:** is this verbose enough? It probably is... but it could be something like "Spawn a node process to run your application" but I thought the extra words might add confusion rather than clarity!

Reply:
I can't think of anything else to add so…

**praise:** Great short clear explanation without fluff!